deployment/docker-compose.override.example.yml

```diff
@@ -52,8 +53,9 @@ services:
   devweb:
     image: kartoza/project-uwsgi:dev-latest
     build:
-      context: docker
-      dockerfile: Dockerfile-dev
+      context: ${PWD}/../
+      dockerfile: deployment/docker/Dockerfile
+      target: dev
```

Comment:
Refactored the Dockerfile-dev to be merged inside Dockerfile file to use multi-stage build.

Specify the build target `dev`